plcc/src/plcc/Main.java

```diff
@@ -974,7 +974,7 @@ else if(args[i+2].equals("remove")) {
                     }
                     
                     if(s.equals("--no-ptgl")) {
-                        Settings.set("plcc_B_strict_ptgl_behaviour", "false");
+                        Settings.set("plcc_B_round_coordinates", "false");
```

Comment:
Setting strict_ptgl meant no rounding. This means that --no-ptgl should set round_coordinates to true if I am correct.